plugins/bufferlines/bufferline.nix

```diff
@@ -11,9 +11,15 @@ with lib; let
   basePluginPath = ["plugins" "bufferline"];
 
   highlight = helpers.mkCompositeOption "Highlight option" {
-    guifg = helpers.mkNullOrOption types.str "foreground color";
+    fg = helpers.mkNullOrOption types.str "foreground color";
 
-    guibg = helpers.mkNullOrOption types.str "background color";
+    bg = helpers.mkNullOrOption types.str "background color";
+
+    sp = helpers.mkNullOrOption types.str "sp color";
+
+    bold = helpers.mkNullOrOption types.bool "enable bold";
+
+    italic = helpers.mkNullOrOption types.bool "enable italic";
```

Comment:
Aren't we missing the mkRenamedOptionModule ? Or are the options completely deleted from upstream ?

Reply:
Finally done !